File: Aula 4/pt 1 desvio simples/DesvioSimples.java
import java.util.Scanner;

public class DesvioSimples {
        public static void main(String[] args) {
            Scanner scanner = new Scanner(System.in);
            System.out.print("Digite a quantidade em estoque: ");
            int quantidade = scanner.nextInt();
        
            if (quantidade < 5) {
                System.out.println("AVISO: O estoque deste produto está baixo!");
            }
            
                scanner.close();
        }
}

File: LPI/Aula 4/pt 1 desvio simples/desvio simples.cpp
#include <iostream>
using namespace std;

int main() {
    int quantidade;
    cout << "Digite a quantidade em estoque: ";
    cin >> quantidade;

    if (quantidade < 5) {
        cout << "AVISO: O estoque deste produto está baixo!" << endl;
    
    }

    return 0;

}